src/components/Form/styles.module.scss

```
@@ -0,0 +1,81 @@
+@import "../../reset.module.scss";
+@import "../../container.module.scss";
+
+.form {
+  background: rgba(46, 80, 113, 1);
+  display: flex;
+  h1 {
+    font-size: 37px;
+    text-align: center;
+    margin-bottom: 40px;
+  }
+  input {
+    color: #fafafa;
+    width: 100%;
+    padding: 0 10px;
+    color: black;
+    border-radius: 4px;
+    margin-bottom: 10px;
+    margin-top: 5px;
+    border: none;
+    height: 34px;
+  }
+  button {
+    color: rgba(82, 85, 87, 1);
+    background: #fff;
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    margin: 0;
+    padding: 0 0.7em;
+    height: 2.4em;
+    line-height: 2.4em;
+    border: none;
+    outline: none;
+    border-radius: 3px;
+    font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Helvetica, Arial,
+      sans-serif;
+    font-weight: 500;
+    font-size: 1.05em;
+    cursor: pointer;
+    transition: all 0.2s ease-in-out;
+    width: 100%;
+  }
+}
+
+.form_container {
+  margin: 50px auto;
+  > h1 {
+    line-height: initial;
+  }
+}
+
+.container_flex {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 16px;
+}
+
+.container_inline_first {
```

```suggestion
.inlineFirst {
```
Pode usar camel case aqui, na minha opinião fica mais com cara de JS